packages/javascript/bh-shared-ui/src/views/Explore/ExploreSearch/edgeTypes.tsx

```diff
@@ -92,6 +92,7 @@ export const AllEdgeTypes: Category[] = [
                     ActiveDirectoryRelationshipKind.GoldenCert,
                     ActiveDirectoryRelationshipKind.ADCSESC1,
                     ActiveDirectoryRelationshipKind.ADCSESC3,
+                    ActiveDirectoryRelationshipKind.ADCSESC4,
```

Comment:
This is the actual functionality change.

Reply:
a handsome change indeed! post processing is awaiting us here: https://github.com/SpecterOps/BloodHound/pull/439